src/Components/Main/NavBar/index.jsx

```diff
@@ -28,14 +32,42 @@ const mapStateToProps = (state) => {
 };
 
 class NavBar extends Component {
+  destPage = {
+    feed: 'feed',
+    funding: 'funding',
+    favorite: 'favorite',
+    shop: 'shop',
+    cart: 'cart',
+    userInfo: 'user',
+    new: 'new',
+  };
+
   constructor(props) {
     super(props);
     this.state = {};
   }
 
-  handleUserPage = () => {
+  getNavLink = (isActive, dest, linkName) => {
+    return (
```

Comment:
isActive는 props로 받아올 수 있으니까 빼고,
dest랑 linkName이랑 똑같이 두면 파라미터 한 개로 줄 것 같지 않나용

```suggestion
  getNavLink = (dest) => {
    const { isActive } = this.props;
    return (
```

Reply:
```javascript
getNavLink = (dest, linkName) => {
    const isActive = this.props;
    return (
      <NavLink
        href="#"
        active={isActive === dest}
        onClick={() => {
          this.handlePage(dest);
        }}
      >
        {linkName}
      </NavLink>
    );
  };
```

linkName을 dest랑 항상 통일 시킬 수가 없어서 isActive만 제거했습니당